### ISSUE TITLE
Correction to getRandomNumber(min,max) function

### ISSUE THREAD

**Issue by susanBuck** [susanBuck/psy1903-forum#10]
Early on we learned the following JavaScript code will yield a random number between 1 and 10:

```js
Math.floor(Math.random() * 10) + 1;
```

When we learned about functions, we used this code in our _getRandomNumber_ function:
```js
function getRandomNumber() {
    return Math.floor(Math.random() * 10) + 1;
}
```

When I expanded this function to discuss adding parameters ([Week 4, Part 1 video](https://harvard.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=8777ca5c-18e0-44cc-9802-b1f00177d511)), I hastily assumed the numbers 10 and 1 were just the max/min and could be swapped out with your dynamic max/min parameters like so:

```js
function getRandomNumber(min, max) { 
    return Math.floor(Math.random() * max) + min;
}
```

Structurally/syntax wise, this code is fine, and everything I said about using parameters with functions still applies, BUT, it will not always produce the expected results (a number between the min and max).

The reason is because in the first formula, the 1 is not actually representing the min. It's used because `Math.floor(Math.random() * 10)` gives a result between 0 and 9 and adding 1 shifts this range to between 1 and 10. This ensures that the result is always a number from 1 to 10.

**Long story short, here is the corrected function and formula for calculating a random number between some dynamic min and max:**

```js
function getRandomNumber(min, max) { 
    return Math.floor(Math.random() * (max - min + 1)) + min;
}
```

How it works:

1. `Math.random()` generates a random floating-point number between 0 (inclusive) and 1 (exclusive).
2. `Math.random() * (max - min + 1)` scales this random number to be between 0 and (max - min), ensuring the range covers the difference between min and max.
3. `ath.floor()` rounds the scaled number down to the nearest whole number.
4. `+ min` shifts the range to start from the min value, giving a final result between min and max (inclusive).
